### DISCUSSION TITLE
MacOS: Right-click in dock for new window

### DISCUSSION THREAD

**Discussion by cburkins** [Eugeny/tabby#6037] [Q&A]
On MacOS, would it be possible to get a right-click menu item for "New Window" so that I could create a new Window in the desktop where I'm currently located ?   

As it stands right now, I have to click on Tabby, which navigates me back to one of the desktops where Tabby already has a window.  From there, I can create a new window, then I have to move that window back to the desktop where I was working (which takes me a second to remember/find).

Small feature, but would be a nice one.

**Comment by binhotvn** [comment]
INDEED!

**Comment by cburkins** [comment]
Perhaps there's a keyboard shortcut that I could send to Tabby to open a new window in my current MacOS "space" (or virtual desktop) ?
